src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,86 @@
+package duke;

```

Comment:
Think it might be nicer to think of a name for the chatbot, I saw that the prof said somewhere that should have been done in level-0 but don't know if will penalise